### PR TITLE
PIGEON-4-implement-routes-for-joining-rooms

### PR DESCRIPTION
**Pull request by duckwarrior103**
Add Routes and placeholder components

### CODE REVIEW

**Review by jackr03 — APPROVED** [review]

**Comment by jackr03** [comment]
Looks good to me!

File: src/App.tsx
Comment (jackr03) [inline src/App.tsx:1]: Do we need this extra <> fragment wrapping the <div>?

File: src/App.tsx [hunk 1]
```diff
@@ -1,10 +1,14 @@
 import './App.css'
-
+import { Outlet} from "react-router-dom";
 function App() {
 
   return (
-    <h1 className='p-4 bg-blue-500 text-white font-bold'>Test</h1>
-  )
-}
+      <>
```
Comment (jackr03) [inline src/App.tsx:6]: Do we need this extra <> fragment wrapping the div?

Comment (duckwarrior103) [reply src/App.tsx:6]: Fuck you lil bro

File: src/routes/EnterCode.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import NavBar from "./NavBar.tsx";
+
+export default function EnterCode() {
+    return (
+        <>
+            <NavBar/>
```
Comment (jackr03) [inline src/routes/EnterCode.tsx:6]: Need a space here

File: src/routes/Room.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import NavBar from "./NavBar.tsx";
+
+export default function Room() {
+    return (
+      <>
+          <NavBar/>
```
Comment (jackr03) [inline src/routes/Room.tsx:6]: Similar to above

File: src/routes/ErrorPage.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+export default function ErrorPage() {
+    return (
+        <>
```
Comment (jackr03) [inline src/routes/ErrorPage.tsx:3]: Similar to above

File: src/routes/EnterCode.tsx (hunk 2 above)
Comment (duckwarrior103) [reply src/routes/EnterCode.tsx:6]: fuck u lil bro

**Pull request merged**